### PR TITLE
Add input result error

### PR DESCRIPTION
**Pull request by iredjee**

### CODE REVIEW

File: src/models/v2/SendInputResponse.ts [hunk 1]
```diff
@@ -1,6 +1,13 @@
+interface InputError {
+  type: string;
+  code: string;
+  message?: string;
+  details?: any;
+}
+
 export class InputResult {
   result: boolean;
+  error: InputError;
```
Comment (vedi) [inline src/models/v2/SendInputResponse.ts]: You just need to make it optional, and remove `Partial` in the next lines.

Comment (iredjee) [reply src/models/v2/SendInputResponse.ts]: Nope, it's tricky, we have 2 possible responses: either `{ result }` or `{ error }`. That's why used `Partial`

Comment (vedi) [reply src/models/v2/SendInputResponse.ts]: So, it mean both are optional. I do not see a reason to declare a type, but use it in a modified way only.

Comment (iredjee) [reply src/models/v2/SendInputResponse.ts]: Updated as agreed

**Pull request merged**